native/html_sanitizer/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+use wasm_bindgen::prelude::*;
+use ammonia::Builder;
+use maplit::{hashset, hashmap};
+
+/// Takes the given HTML string and removes anything not allowed.
+/// Preserves inline styles on div, p, a, img, and h1-h6 tags.
+/// ## Arguments
+///
+/// * `html_string` - A string containing HTML to be sanitized.
+#[wasm_bindgen]
+pub fn sanitize_html(html_string: &str) -> String {
+  Builder::default()
+    //Allow inline styles on <div>, <p>, <a>, <img>, and the <hX> tags
+    .add_tag_attributes("div", &["style"])
+    .add_tag_attributes("p", &["style"])
+    .add_tag_attributes("a", &["style"])
+    .add_tag_attributes("h1", &["style"])
+    .add_tag_attributes("h2", &["style"])
+    .add_tag_attributes("h3", &["style"])
+    .add_tag_attributes("h4", &["style"])
+    .add_tag_attributes("h5", &["style"])
+    .add_tag_attributes("h6", &["style"])
+    .add_tag_attributes("img", &["style"])
```

Comment:
We can probably customize this a lot more. I predict we'll need to, in order to support the way Froala styles inline images.